test/board_test.exs

```diff
@@ -0,0 +1,28 @@
+defmodule BoardTest do
+  use ExUnit.Case
+  doctest TicTacToeElixir
+
+  test "returns blank 3x3 board" do
+    assert Board.print_board(nil) == "123\n456\n789"
```

Comment:
**thought:** Could there be any downsides of you attributing some semantic domain meaning to a `nil` in this instance?